.github/workflows/go.yml

```
@@ -30,17 +30,20 @@ jobs:
           key: ${{ runner.os }}-go-${{ matrix.go }}-${{ hashFiles('tests/go.mod') }}
 
       - name: Build
-        run: CGO_ENABLED=0 find . -name go.mod -execdir go build ./... \;
+        run: find . -name go.mod -execdir sh -c 'pwd && CGO_ENABLED=0 go build ./... || exit 1' sh {} +
```

Can't this just be
```suggestion
        run: CGO_ENABLED=0 go build ./...
```
? Does it need to build independently in each module?

---

It does. `go build ./...` does not seem to build the the sub-directories.

You can tell because `GOOS=js GOARCH=wasm go build ./...` works, but `cd plugin && GOOS=js GOARCH=wasm go build ./...` fails (correctly).